src/main/java/com/google/sps/servlets/MentorApprovalServlet.java

```diff
@@ -120,11 +137,115 @@ private String getMentorEvidence(int mentorId) {
+    // Create the MySQL prepared statement.
+    String query = "UPDATE MentorApproval "
+        + "SET is_reviewed = TRUE "
+        + "WHERE mentor_id = " + Integer.toString(mentorId)
+        + " AND approver_id = " + Integer.toString(approverId);
+    
+    try {
+      // Establish connection to MySQL database.
+      Connection connection = DriverManager.getConnection(
+          Utility.SQL_LOCAL_URL, Utility.SQL_LOCAL_USER, Utility.SQL_LOCAL_PASSWORD);
+      
+      // Execute the MySQL SELECT prepared statement.
+      PreparedStatement preparedStatement = connection.prepareStatement(query);
+      preparedStatement.execute();
+      connection.close();
+    } catch (SQLException exception) {
+      // If the connection or the query don't go through, we get the log of what happened.
+      Logger logger = Logger.getLogger(MentorApprovalServlet.class.getName());
+      logger.log(Level.SEVERE, exception.getMessage(), exception);
+    }
+  }
+
+  /**
+   * Updates is_approved, is_rejected or approvals variables in MentorEvidence table based on
+   * approver review.
+   */
+  private void addEvidence(boolean isApproved, int mentorId) {
+    // Get current number of approvals mentor has.
+    int numberOfApprovals = getNumberOfApprovals(mentorId);
+    
+    // Create the MySQL prepared statement.
+    String query = "";
+    if(isApproved && numberOfApprovals == 1) {
+      // If user is approved by approver, and already has one approval,
+      // increment number of approvals and update is_approved in MentorEvidence table.
+      query = "UPDATE MentorEvidence "
+          + "SET approvals = 2, is_approved = TRUE "
+          + "WHERE mentor_id = " + Integer.toString(mentorId);
+    } else if (isApproved && numberOfApprovals == 0) {
+      // If user is approved by approver, but has no previous approvals,
+      // increment number of approvals in MentorEvidence table.
+      query = "UPDATE MentorEvidence "
+          + "SET approvals = 1 "
+          + "WHERE mentor_id = " + Integer.toString(mentorId);
+    } else {
+      // If user is rejected, update is_rejected in MentorEvidence table.
+      query = "UPDATE MentorEvidence "
+          + "SET is_rejected = TRUE "
+          + "WHERE mentor_id = " + Integer.toString(mentorId);
+    }
+    
+    try {
+      // Establish connection to MySQL database.
+      Connection connection = DriverManager.getConnection(
+          Utility.SQL_LOCAL_URL, Utility.SQL_LOCAL_USER, Utility.SQL_LOCAL_PASSWORD);
+      
+      // Execute the MySQL SELECT prepared statement.
+      PreparedStatement preparedStatement = connection.prepareStatement(query);
+      preparedStatement.execute();
+      connection.close();
+    } catch (SQLException exception) {
+      // If the connection or the query don't go through, we get the log of what happened.
+      Logger logger = Logger.getLogger(MentorApprovalServlet.class.getName());
+      logger.log(Level.SEVERE, exception.getMessage(), exception);
+    }
```

Comment:
You could probably make this into a Utility function (called something along the lines of "ExecuteQuery") - that just takes in a string query and attempts to execute it. It looks like this logic is shared several times throughout this file and I imagine it could be useful in other servlets as well!